### PR TITLE
amazon s3 resource plugin

### PR DESCRIPTION
**Pull request by scott-cotton**

### CODE REVIEW

**Review by Freezerburn — APPROVED** [review]
LGTM

File: src/amazon-s3/provision [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+if [[ -e "${INPUT_DIR}/args" ]]; then
+  ATTRIBUTES=$(cat "${INPUT_DIR}/args")
+  echo "args: ${ARGS}"
+  EXTRA_ARGS+=("${ARGS}")
+fi
+
+
+# Choose a bucket name that's globally unique.  The sandbox id
+# and prefix -signadot should guarantee uniqueness with high probability.
```
[the remark this answers is not in the document]
Comment (scott-cotton) [reply src/amazon-s3/provision]: Thanks! fixed.

File: signadot-plugins-exp/amazon-s3/README.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+helm -n signadot install amazon-s3-plugin signadot-plugins-exp/amazon-s3
+```
+
+### Configuring the Plugin
+
+The following values can be configured for this chart via `--set` or `values.yaml`:
+
+Value | Description | Default
+----- | ----------- | -------
+`image` | The container image URL for the plugin | `signadot/amazon-sqs-plugin:{version}`
```
Comment (foxish) [inline signadot-plugins-exp/amazon-s3/README.md:34]: `amazon-s3-plugin`

**Review by foxish — APPROVED** [review]
LGTM, thanks! Just the one comment.

File: signadot-plugins-exp/amazon-s3/README.md (hunk 2 above)
Comment (scott-cotton) [reply signadot-plugins-exp/amazon-s3/README.md]: thanks for the review and the catch!

**Pull request merged**